components/rhel/rhel-ose/Vagrantfile

```diff
@@ -64,6 +54,26 @@ Vagrant.configure(2) do |config|
     config.registration.password = ENV['SUB_PASSWORD']
   end
 
+  if ENV.has_key?('PROXY')
```

Comment:
You can try it in one line as :

```
  config.registration.proxy = PROXY = (ENV['PROXY'] || '')
  # other similar variables
```
In above case, lines from 73 to 75 are not required.